hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestBucketList.java

```diff
@@ -525,14 +452,12 @@ private void assertEncodingTypeObject(
     assertEquals(exceptEncodingType, object.getEncodingType());
   }
 
-  private OzoneClient createClientWithKeys(String... keys) throws IOException {
-    OzoneClient client = new OzoneClientStub();
-
-    client.getObjectStore().createS3Bucket("b1");
-    OzoneBucket bucket = client.getObjectStore().getS3Bucket("b1");
+  private void createClientWithKeys(String... keys) throws IOException {
```

Comment:
I would prefer to keep the return here, then assign the returned client to the `this.client` in setup.

Reply:
or
```suggestion
  private void createKeys(Client client, String... keys) throws IOException {
```
and the setup:

```
setup (keys) {
   client = new client;
   createKeys(client, keys);
}
```